quicklogic/common/cmake/quicklogic_install.cmake

```diff
@@ -63,10 +63,19 @@ function(DEFINE_QL_TOOLCHAIN_TARGET)
   install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/counter_16bit_tb.v
           DESTINATION share/symbiflow/tests/counter_16bit
           PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
+  install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/counter_16bit.pcf
+	  DESTINATION share/symbiflow/tests/counter_16bit
+          PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
+  install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/pinmap_qlf_k4n8_umc22.csv
+	  DESTINATION share/symbiflow/tests/counter_16bit
+	  PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
   install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/counter_16bit.sdc
           DESTINATION share/symbiflow/tests/counter_16bit
           PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
-  
+  install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/clk_constraints.json
+          DESTINATION share/symbiflow/tests/counter_16bit
+          PERMISSIONS WORLD_READ OWNER_WRITE OWNER_READ GROUP_READ)
+
```

Comment:
I'm sure that installation of the `counter_16bit` example for the installed toolchain should not be done here.

Reply:
Hmm, yeah... I think we might need to add a function to install examples tests and use that one.

Reply:
true. This we added during the release, so that customer can have alteast one test design to start with.

Reply:
Ok, I think that, for the time being, we can leave the installation of the example out of this PR and add it properly once everything lands upstream.